twindle-web/team_details/team_details.js

```diff
@@ -1,52 +1,120 @@
+         twitter: "https://twitter.com/rafa_55",
+      },
+      title: "FrontEnd Developer",
+      location: {
+         city: "Resende",
+         state: "Rio de Janeiro",
+         country: "Brazil",
+      },
+    },
+    ,{
+      id:4,
+      name: "Ameen Shafeeq",
+      img: "https://avatars0.githubusercontent.com/u/49345531?v=4",
+      links: {
+        website: "https://ameencodes.tech/",
+        linkedin: "",
+        github: "https://github.com/unevencoder",
+        twitter: "https://twitter.com/crafter_coder",
+      },
+      title: "Student",
+      location: {
+        city: "Jeddah",
+        state: "",
+        country: "Saudi Arabia"
+      }
+    }
+  ]
 
 const cards = document.querySelector(".card-gallery");
 
-function generateCard() {
+function generateCard(name,img,website,linkedin,github,twitter,title,city,state,country) {
     return `
     <div class="card">
         <div class="card-heading">
           <img
-            src="https://avatars0.githubusercontent.com/u/37118877?v=4"
+            src=`+img+`
             class="image"
           />
-          <h2 class="name">Akshay Sharma</h2>
-          <h5 class="title">Front End Developer</h5>
+          <h2 class="name">`+name+`</h2>
+          <h5 class="title">`+title+`</h5>
         </div>
         <hr />
         <div class="card-links">
           <a
-            href="https://github.com/akshay2996"
+            href=`+github+`
             target="_blank"
             rel="noopener noreferrer"
             ><i class="fa fa-github fa-2x" aria-hidden="true"></i
           ></a>
           <a
-            href="https://twitter.com/akshay2996"
+            href=`+twitter+`
             target="_blank"
             rel="noopener noreferrer"
             ><i class="fa fa-twitter fa-2x" aria-hidden="true"></i
           ></a>
-          <a href="linkedin" target="_blank" rel="noopener noreferrer"
+          <a href=`+linkedin+` target="_blank" rel="noopener noreferrer"
             ><i class="fa fa-linkedin fa-2x" aria-hidden="true"></i
           ></a>
           <a
-            href="https://developeratease.com/"
+            href=`+website+`
             target="_blank"
             rel="noopener noreferrer"
             ><i class="fa fa-globe fa-2x" aria-hidden="true"></i
           ></a>
         </div>
         <div class="card-location">
-          <h5 class="location">Bangalore, Karnataka, India</h5>
+          <h5 class="location">`+city+` `+state+` `+country+`</h5>
         </div>
       </div>
     `;
 }
 
-const cardArray = Array.from({length: 7});
-
-const html = cardArray.map(generateCard).join("");
-// console.log(html);
-cards.innerHTML = html;
-// console.log(cards.innerHTML);
\ No newline at end of file
+data.forEach( data =>cards.innerHTML+=generateCard(data.name,data.img,data.links.website,data.links.linkedin,data.links.github,data.links.twitter,data.title,data.location.city,data.location.state,data.location.country)
```

Comment:
It may be more performant to collect everything as string and do single innerHTML. Just consider if it is needed.